blocks.js

```diff
@@ -7553,13 +7554,38 @@ InputSlotMorph.prototype.dropDownMenu = function (enableKeyboard) {
         if (Object.prototype.hasOwnProperty.call(choices, key)) {
             if (key[0] === '~') {
                 menu.addLine();
-            // } else if (key.indexOf('§_def') === 0) {
-            //     menu.addItem(choices[key].blockInstance(), choices[key]);
+        // } else if (key.indexOf('§_def') === 0) { 
+        // menu.addItem(choices[key].blockInstance(), choices[key]);
+            } else if (key.charCodeAt(key.length-1) == 0x25ba) {
+                // Submenu
+                menu.addItem(key, choices[key]);
+                menustack.push(menu);
+                menu = new MenuMorph(
+                                     this.setContents,
+                                     null,
+                                     this,
+                                     this.fontSize
+                                 );
+            } else if (key.charCodeAt(0) == 0x25c4) {
+                var oldmenu=menustack[menustack.length-1],
```

Comment:
I was being sarcastic with snake case. Lol. 

Even if you have poor descisions it's good to write consistent code or at least declare which code isn't going to follow conventions and why. 

I really ought to revive my ESlint branch.